From 066f23689bf4b417bf82895aa37d8d5a00e704bf Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Mon, 20 Jan 2014 12:49:35 +0100
Subject: [PATCH 1/4] help: add id suboption to -iscsi

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1390222175-24815-1-git-send-email-pbonzini@redhat.com>
Patchwork-id: 56830
O-Subject: [RHEL 7.0 qemu-kvm PATCH] help: add id suboption to -iscsi
Bugzilla: 1019221
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Bugzilla: 1019221

Brew build: 6896455

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 2fe3798cd5fab65ee7c86758b1b7701d7fe3709f)
---
        As requested by QE.

 qemu-options.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-options.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index 49b4bc2..1f2bb57 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -2073,7 +2073,7 @@ ETEXI
 DEF("iscsi", HAS_ARG, QEMU_OPTION_iscsi,
     "-iscsi [user=user][,password=password]\n"
     "       [,header-digest=CRC32C|CR32C-NONE|NONE-CRC32C|NONE\n"
-    "       [,initiator-name=iqn]\n"
+    "       [,initiator-name=initiator-iqn][,id=target-iqn]\n"
     "                iSCSI session parameters\n", QEMU_ARCH_ALL)
 STEXI
 
-- 
1.7.1

